lib/Importer/ONNXModelLoader.cpp

```diff
@@ -529,6 +529,17 @@ bool ONNXModelLoader::loadOperator(const ONNX_NAMESPACE::NodeProto &op) {
     return true;
   }
 
+  if (typeName == "SparseToDense") {
+    auto indices = getNodeValueOrCreateConstantByName(op.input(0));
+    auto values = getNodeValueOrCreateConstantByName(op.input(1));
+    auto dataToInferDim = getNodeValueOrCreateConstantByName(op.input(2));
+
+    auto *node =
+        G_.createSparseToDense(opName, indices, values, dataToInferDim);
+    addNodeAsOutput(op, node);
+    return true;
+  }
+
```

Comment:
This one is the same implementation as Caffe2. So could be put into CommonOperatorLoader.h

Reply:
Hmm, I added a small if-statement to fail the loading if the number of inputs is not correct because `dataToInferDim` is an optional argument in Caffe2 but required for Glow (because of dynamic shapes :( ). So I will leave this here for now.